Patches/MeetingHudPatch.cs

```diff
@@ -1055,6 +1118,20 @@ public static void Postfix(MeetingHud __instance)
                 roleTextMeeting.text += TaskState.GetTaskState(); // Random task count for revealed trickster
                 //enable = false;
             }
+            if (!PlayerControl.LocalPlayer.Data.IsDead && Overseer.IsRevealedPlayer(PlayerControl.LocalPlayer, pc) && Illusionist.IsCovIllusioned(pc.PlayerId))
```

Comment:
there's no method for IsAlive on PlayerControl.LocalPlayer.Data
(also because it's just a carbon copy of the code above it it does the same exact thing)